scipy/stats/_qmc.py

```diff
@@ -1701,7 +1715,9 @@ def __init__(
         if not np.isclose(np.sum(pvals), 1):
             raise ValueError('Elements of pvals must sum to 1.')
         if engine is None:
-            self.engine = Sobol(d=1, scramble=True, seed=seed)  # type: QMCEngine
+            self.engine = Sobol(
+                d=1, scramble=True, bits=30, seed=seed
```

Comment:
Why specify `bits=30` here? If that is the default, why not `None` or just leave it unspecified?

Reply:
I changed this because locally I started to think about using a deprecation warning in case of `None`. So I did not want to raise a warning in such cases (I think)